py-polars/tests/unit/sql/test_temporal.py

```diff
@@ -86,6 +86,7 @@ def test_datetime_to_time(time_unit: Literal["ns", "us", "ms"]) -> None:
         ),
     ],
 )
+@pytest.mark.skip(reason="don't understand; will ask @alex")
```

Comment:
@alexander-beedie could you take a look at this one? I don't really understand what causes this.

Reply:
Fixed: https://github.com/pola-rs/polars/pull/15955.